File: lektion-1/cpp/program.cpp
#include <iostream>
#include <ctime>

using namespace std;

int main() 
{
    std::string name;
    std::cout << "Vad heter du? ";
    std::cin >> name;

    int age;
    cout << "\nHur gammal är du? ";
    cin >> age;

    time_t current_time = time(NULL);
    int current_year;
    current_year = 1970 + current_time / 31537970;
    
    int birth_year;
    birth_year = current_year - age;
    cout << "Okej " << name << ". Du föddes alltså år " << birth_year << ".";

    string has_children;
    cout << "\nHar du några barn? ";
    cin >> has_children;

    if (has_children == "ja") 
    {
        int children_amount;
        cout << "\nHur många barn har du? ";
        cin >> children_amount;
        cout << "Trevligt med " << children_amount << " barn. Själv har jag inga barn då jag är ett program.";
    }
    else 
    {
        cout << "Inte jag heller, för att jag är ett program.";
    }    

    return 0;
}

File: lektion-1/cs/Program.cs
string name;
Console.Write("Vad heter du? ");
name = Console.ReadLine();

int age;
Console.Write("Hur gammal är du? ");
age = int.Parse(Console.ReadLine());

int current_year;
current_year = DateTime.Now.Year;

int birth_year = current_year - age;
Console.WriteLine($"Okej {name}. Du föddes alltså år {birth_year}.");

string has_children; 
Console.Write("Har du några barn? ");
has_children = Console.ReadLine();

if (has_children.ToLower() == "ja") 
{    
    int children_amount; 
    Console.Write("Hur många barn har du? ");
    children_amount = int.Parse(Console.ReadLine());
    Console.WriteLine($"Trevligt med {children_amount} barn. Själv har jag inga barn då jag är ett program.");
}
else
{
    Console.WriteLine("Inte jag heller, för att jag är ett program.");
}
    